taskcluster/rb_taskgraph/job.py

```diff
@@ -75,10 +75,10 @@ def configure_gradlew(config, job, taskdesc):
     run = job["run"]
     worker = taskdesc["worker"] = job["worker"]
 
+    fetches_dir = path.join(run["workdir"], worker["env"]["MOZ_FETCHES_DIR"])
     worker.setdefault("env", {}).update({
-        "ANDROID_SDK_ROOT": path.join(
-            run["workdir"], worker["env"]["MOZ_FETCHES_DIR"], "android-sdk-linux"
-        )
+        "ANDROID_SDK_ROOT": path.join(fetches_dir, "android-sdk-linux"),
+        "GRADLE_USER_HOME": path.join(fetches_dir, "android-gradle-dependencies"),
```

Comment:
I'm not sure setting this to point into the fetches directory makes sense; it feels like an expedient solution that willl cause surprise in the future (largely because I `GRADLE_USER_HOME` is writable, and I would not necessarily expect fetches to be modified). I'm curious what m-c does with the gradle-depndencies package

Reply:
I agree this may be a problem in the future. m-c exposes `android-gradle-dependencies` as a maven repository, so gradle doesn't write in it, it just copies files from that folder to `caches/modules-2/files-2.1/`. I think we want eventually want to go this way. I'll address it in a followup.